website/project/views/node.py

```diff
@@ -735,6 +736,13 @@ def _view_project(node, auth, primary=False,
             'institutions': get_affiliated_institutions(node) if node else [],
             'has_draft_registrations': node.has_active_draft_registrations,
             'is_preprint': node.is_preprint,
+            'has_moderated_preprint': node.has_moderated_preprint,
```

Comment:
Instead of `node.has_moderated_preprint`, could this check `node_linked_preprint.provider.reviews_workflow`?